### PR TITLE
Readable format

### PR DESCRIPTION
**Pull request by tschortsch**

### CODE REVIEW

File: ckanext/switzerland/plugin.py [hunk 1]
```diff
@@ -227,6 +232,70 @@ def _package_map_ckan_default_fields(self, pkg_dict):  # noqa
...
+            url = resource['download_url']
+            path = urlparse.urlparse(url).path
+            ext = os.path.splitext(path)[1]
+            if ext:
+                resource_format = ext.replace('.', '').lower()
+                resource['format'] = self._map_to_valid_format(resource_format)
+                return resource
+
+        # get format from media_type field if available
+        if 'media_type' in resource and resource['media_type']:
```
Comment (metaodi) [inline ckanext/switzerland/plugin.py]: use `if not resource_format and and resource.get('media_type') is not None:` and get rid of the return statement

File: ckanext/switzerland/plugin.py [hunk 2]
```diff
@@ -227,6 +232,70 @@ def _package_map_ckan_default_fields(self, pkg_dict):  # noqa
...
+            resource_format = resource['media_type'].split('/')[-1].lower()
+            resource['format'] = self._map_to_valid_format(resource_format)
+            return resource
+
+        # get format from format field if available (lol)
+        if 'format' in resource and resource['format']:
```
Comment (metaodi) [inline ckanext/switzerland/plugin.py]: Dito, see above

File: ckanext/switzerland/plugin.py [hunk 3]
```diff
@@ -227,6 +232,70 @@ def _package_map_ckan_default_fields(self, pkg_dict):  # noqa
...
+
+        # get format from download_url file extension if available
+        if 'download_url' in resource and resource['download_url']:
+            url = resource['download_url']
+            path = urlparse.urlparse(url).path
+            ext = os.path.splitext(path)[1]
+            if ext:
+                resource_format = ext.replace('.', '').lower()
+                resource['format'] = self._map_to_valid_format(resource_format)
+                return resource
```
Comment (metaodi) [inline ckanext/switzerland/plugin.py]: get rid of last two lines

File: ckanext/switzerland/plugin.py [hunk 4]
```diff
@@ -152,6 +154,9 @@ def _prepare_package_json(self, pkg_dict):
         # map ckan fields
         pkg_dict = self._package_map_ckan_default_fields(pkg_dict)
 
+        # prepre format of resources
```
Comment (stefina) [inline ckanext/switzerland/plugin.py]: typo? you probably meant prepare?

**Comment by stefina** [comment]
Please invert the assertEqual-Statement to use first the expected, then the actual value, to make it conform.

**Pull request merged**